api/src/pcapi/core/offers/offer_metadata.py

```diff
@@ -72,6 +80,50 @@ def _get_event_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
     return common_metadata | event_metadata
 
 
+def _get_book_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
+    common_metadata = _get_common_metadata_from_offer(offer)
+
+    book_metadata = {
+        "@type": "Book",
+        "@id": offer_app_link(offer),
+        "isFamilyFriendly": not offer.is_forbidden_to_underage,
+        "url": offer_app_link(offer),
+    }
+
+    work_example = {
+        "@type": "Book",
+        "@id": offer_app_link(offer),
+        "inLanguage": "fr",
+    }
+
+    extra_data = offer.extraData
+    if extra_data:
+        author = extra_data.get("author")
+        ean = extra_data.get("ean")
+        if author:
+            book_metadata["author"] = {
+                "@type": "Person",
+                "name": author,
+            }
+        if ean:
+            work_example["isbn"] = ean
```

Comment:
[détails / style] Pour être un peu plus concis (sans perdre en lisibilité), tu peux utilise l'opérateur `:=` (appelé *walrus operator*, apparu en Python 3.8) : 

```suggestion
        if author := extra_data.get("author"):
            book_metadata["author"] = {
                "@type": "Person",
                "name": author,
            }
        if ean := extra_data.get("ean"):
            work_example["isbn"] = ean
```

Reply:
Trop stylé cet opérateur j'en avais jamais entendu parlé